ada/views.py

```diff
@@ -33,9 +33,11 @@ def clickActivity(request: HttpRequest, pk: int, whatsappid: str) -> HttpRespons
     else:
         store_url_entry = RedirectUrlsEntry(symptom_check_url=redirect_url)
         store_url_entry.save()
+        customizationId = "kh93qnNLps"
         url = f"{redirect_url.symptom_check_url}"
-        qs = urlencode({"whatsappid": whatsappid})
-        destination_url = f"{url}?{qs}"
+        whatsapp_qs = urlencode({"whatsappid": whatsappid})
```

Comment:
Is it not possible to urlencode this all in one step?

Reply:
Might be. Let me see